session/session.go

```diff
@@ -105,7 +110,11 @@ func New(
 		HTTPNoProxyClient: httpx.NewTracingProxyingClient(
 			logger, nil, tlsConfig,
 		),
-		Logger:          logger,
+		Logger: logger,
+		PrivacySettings: collector.PrivacySettings{
+			IncludeCountry: true,
+			IncludeASN:     true,
+		},
```

Comment:
In this constructor is where you could pass a reference to `Location` see: https://github.com/ooni/probe-engine/pull/37/files#r313414579

Reply:
As I mentioned, I'm not 100% sold on using location because in theory you can run tests with the collector disabled and then attempt to submit later. So, refer to the above discussion regarding how I am planning on dealing with this situation. I am going to close this conversation as we'd otherwise have two redundant branches.